Sources/Widgets/Widgets3D/ResliceCursorWidget/example/index.js

```diff
@@ -405,7 +431,7 @@ reader.setUrl(`${__BASE_PATH__}/data/volume/LIDC2.vti`).then(() => {
                 renderer: obj.renderer,
                 resetFocalPoint: false,
                 keepFocalPointPosition,
-                computeFocalPointOffset,
+                computeFocalPointOffset: !keepFocalPointPosition,
```

Comment:
It looks like `keepFocalPointPosition` is always `!computeFocalPointOffset`, does it make sens to keep both variables?
It would make sense to say that if we don't keep focal position, then we always need to compute focal point offset